react/features/app/getRouteToRender.native.js

```diff
@@ -30,10 +31,8 @@ function _getMobileRoute(state) {
 
     if (isRoomValid(state['features/base/conference'].room)) {
         route.component = ConferenceNavigationContainer;
-    } else if (isWelcomePageAppEnabled(state)) {
-        route.component = WelcomePage;
     } else {
-        route.component = BlankPage;
+        route.component = RootNavigationContainer;
```

Comment:
You can rework this now, since `getEmptyRoute` is unnecessary.

Reply:
Updated.